pages/content/amp-dev/documentation/guides-and-tutorials/develop/interactivity_guide/index.md

```diff
@@ -0,0 +1,104 @@
+---
+formats:
+  - websites
+$title: Interactivity in AMP introduction
+$order: 0
+description: "Read this guide to learn how to build interactivity in AMP."
+$hidden: true
+author: CrystalOnScript
+---
+
+These guides outline how to create interactive AMP pages. Interactivity is essential to the modern web. Building unique and delightful experiences engages users, keeping them on your site. AMP provides out-of-the box solutions to common interactivity needs while also affording highly customizable and flexible solutions.
+
+Building interactive experiences in AMP may feel untraditional, and it is! But that doesn’t mean it’s more difficult. Many interactive experiences are very easy to build with AMP once you’re introduced to concepts and understand the syntax.
```

Comment:
Optional: Something about the "It is <not traditional>" is a bit confusing for me, can we instead rephrase to say "It is not <traditional>", i.e.  `"Building interactive experiences may not feel traditional, and it's not"`

Reply:
I agree - the rewrite is much better. Thanks!